source/concepts.tex

```diff
@@ -300,7 +301,8 @@
 \begin{itemdescr}
 \pnum
 \begin{note}
-\tcode{\libconcept{DerivedFrom}<Derived, Base>} is satisfied if and only if
+Types \tcode{Derived} and \tcode{Base} model
+\tcode{\libconcept{DerivedFrom}<Derived, Base>} if and only if
```

Comment:
What are the semantic requirements here that mean we have to use modelling not satisfaction?

Reply:
This change is for consistency of presentation; false consistency, as you've pointed out. I'll revert.